istioctl/pkg/analyze/analyze.go

```diff
@@ -491,6 +495,9 @@ type Client struct {
 }
 
 func getClients(ctx cli.Context) ([]*Client, error) {
+	if len(remoteClusterContexts) > 0 {
+		return getClientsFromContexts(ctx)
```

Comment:
Here if I only specify remote contexts, the original default one will be missed?

Reply:
I think what you're looking for is covered in L547-L555

Lmk if I misunderstood your comment

Reply:
Yes that's what I'm pointing to. Probably we can put L501-L510 to the front and append the rest of clients to it based on the contexts/secrets.

Reply:
Hmm, makes sense. Refactored it a bit. PTAL.